WordPress/Classes/ViewRelated/Post/PostEditorNavigationBarManager.swift

```diff
@@ -164,7 +255,7 @@ class PostEditorNavigationBarManager {
     }
 
     func reloadBlogTitleView(text: String) {
-        blogTitleViewLabel.text = text
+        blogTitleViewLabel.text = ""
```

Comment:
@geriux Is this an intentional change?

Reply:
Hey @momo-ozawa 👋  Nice catch! With the new design changes we are not displaying the Blog's title in the navigation bar anymore but it was being used by the Aztec editor since they shared the same navigation bar. When I looked into this I realized I needed to do some extra tweaks added in https://github.com/wordpress-mobile/WordPress-iOS/pull/20930/commits/d22dc572bac95ae7b7196903223c7c097bfde21b

## Changing the "Generating Preview" behavior

Currently, we were updating the navigation bar to display when the post preview was being generated, with the new changes there's not much space available so I updated it to be consistent with Android so now it shows like this:

iOS|Android
-|-
<video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/77625f83-d555-421b-ba2f-91f34cbbde64"/> | <video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/732ade80-a84c-4fdf-8e60-c32e1a9622a5" />

## Keeping the old `rightButtonItems` for the Aztec editor

Since the new undo/redo functionality is only available for the Gutenberg editor these shouldn't show for the Aztec editor so I've updated it so it looks like this:

Gutenberg|Aztec
-|-
<kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/87cf331d-1e5e-4b34-bb21-2b0daeca90b3" width="200" /></kbd> | <kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/a5ea4472-6c7b-45a5-8034-d33151aac039" width="200" /></kbd>

## Removal of the `reloadBlogTitleView` logic

Since the title is not shown in either Gutenberg or Aztec editors, I've removed it.

Let me know what you think! Thanks for the review and testing!

Reply:
Makes sense to me! Thank you!